test/test_jit_fuser_te.py

```diff
@@ -2202,7 +2202,7 @@ def test_batch_norm(self):
         def test(fn, args):
             trace = torch.jit.trace(fn, args)
             self.assertAllFused(trace.graph_for(*args))
-            torch.testing.assert_allclose(fn(*args), trace(*args))
+            torch.testing.assert_close(fn(*args), trace(*args), equal_nan=True)
```

Comment:
In `torch.testing.assert_close`, `equal_nan` is `False` by default. Are `NaN`'s actually ok here or did this pass silently before?